blocks/banner/banner.js

```diff
@@ -0,0 +1,76 @@
+import {
+  div, button, h2, h3, form, input, a,
+} from '../../scripts/dom-helpers.js';
+
+function createLinkElement(hrefValue, innerHTML) {
+  const link = a({ class: 'spotlight-link', href: hrefValue }, innerHTML);
+
+  return link;
+}
+
+function createButton(hrefValue) {
+  const spotlightButton = button({
+    class: 'spotlight-button',
+    onclick: () => { window.location.href = hrefValue; },
+  }, 'LEARN MORE');
+
+  return spotlightButton;
+}
+
+function createBannerText(block) {
+  // extract text values from block elements
+  const spotlightTitle = block.querySelector('h2');
+  const spotlightSubtitle = block.querySelector('h3');
+
+  // create hyperlinks with the text values
+  const spotlightLink = block.querySelector('a');
+  const hrefValue = spotlightLink ? spotlightLink.href : '#';
+  const titleLink = createLinkElement(hrefValue, spotlightTitle.innerHTML);
+  const subtitleLink = createLinkElement(hrefValue, spotlightSubtitle.innerHTML);
+
+  spotlightTitle.remove();
+  spotlightSubtitle.remove();
+  spotlightLink.remove();
+
+  // style the hyperlinks
+  const heading = h2({ class: 'spotlight-text title' }, titleLink);
```

Comment:
If the div that contains these h2 and h3 is the `spotlight-container`, then you could simply say:
```
.spotlight-container h2 { } 
.spotlight-container h3 { }
```
There would be no need for the title and the subtitle as the h2 is the title and the h3 is the subtitle.

Reply:
okay, updated.